common-docs/teachertool/catalog-shared.json

```diff
@@ -43,25 +22,69 @@
             "description": "At least one variable's value is read."
         },
         {
-            "id": "8E6F9A92-2D22-4BB0-A77A-BD7490D3CEF7",
-            "use": "variable_set_random",
-            "template": "A custom variable's value is set to a random number",
+            "id": "7AE7EA2A-3AC8-42DC-89DB-65E3AE157156",
+            "use": "block_comment_used",
+            "template": "At least ${count} comments",
```

Comment:
Not for this change, but wondering if we're still planning on updating this validator since it's just looking at block comments right now.

Reply:
Yeah, it'd be good to do. Not highest pri at the moment, though.

Reply:
When I get around to moving all the work items from our task board over to issues/bugs, I'll make sure we include that one.